docs/resource-policy/setup.md

```diff
@@ -0,0 +1,61 @@
+# Setup and Usage
+
+When you want to try NRI Resource Policy, here is the list of things
+you need to do, assuming you already have a Kubernetes\* cluster up and
+running, using either `containerd` or `cri-o` as the runtime.
+
+  * [Install](installation.md) NRI Resource Policy DaemonSet deployment file.
+  * Runtime (containerd / cri-o) configuration
+
+For NRI Resource Policy, you need to provide a configuration file. The default
+configuration ConfigMap file can be found in the DaemonSet deployment yaml file.
+You can edit it as needed.
+
+**NOTE**: Currently, the available policies are a work in progress.
+
+## Setting up NRI Resource Policy
+
+### Using NRI Resource Policy Agent and a ConfigMap
```

Comment:
This stuff could be moved to the node-agent/dynamic configuration document, if not already covered there(?)

Reply:
I suggest we revisit all the setup, quick-start and relevant docs after the helm chart support is merged which is probably quite soon.